joycond-cemuhook.py

```diff
@@ -133,9 +133,15 @@ def __init__(self, server, index, device, motion_device, motion_only=False):
         self.player_id = get_player_id(self.led_status)
 
         with open(os.path.join('profiles', self.name + '.json')) as profile:
-            self.keymap = {evdev.ecodes.ecodes[ecode]:ps_key for ecode,ps_key in json.load(profile).items()}
+            original_keymap = json.load(profile).items()
+            self.keymap = {evdev.ecodes.ecodes[ecode.lstrip('-')]:[] for ps_key,ecode in original_keymap if ecode is not None}
+            for ps_key,ecode in original_keymap:
+                if ecode is not None:
+                    prefix = '-' if ecode.startswith('-') else ''
+                    self.keymap[evdev.ecodes.ecodes[ecode.lstrip('-')]].append(prefix+ps_key)
```

Comment:
I think it would be better to put the minus sign in the `ps_key` side (in the json files), then this bit of code would be simpler (no need to strip `ecode`, and `ps_key` would already contain the prefix).

Reply:
My idea here is that the config files would be based on "plain" cemuhook buttons and axes, mapped to (possibly more complex) joycond buttons and axes. Unless there's another way around this. I think for now I'm merging this PR then to avoid having new users experience this bug!